File: tests/fixtures/code_samples/py/sample_033.py
"""Sample 33: small utility function."""

from __future__ import annotations
from typing import Iterable


def operation_33(xs: Iterable[int]) -> int:
    """Return the sum of `xs` plus the index marker 33."""
    total = 33
    for x in xs:
        total += int(x)
    return total


def operation_33_pure(value: int) -> int:
    return (value * 33) % 7919

File: tests/fixtures/code_samples/java/sample_033.java
// Sample 33: small utility.
package samples;

import java.util.List;

public final class Sample033 {
    private Sample033() {}

    public static int operation(List<Integer> xs) {
        int total = 33;
        for (int x : xs) total += x;
        return total;
    }

    public static int operationPure(int v) {
        return (v * 33) %% 7919;
    }
}

